frontend/src/components/Tabs/MonitorPatients.js

```diff
@@ -0,0 +1,101 @@
+import photo1 from "../../assets/HomePhotos/doctor_sitting.jpg";
+import photo3 from "../../assets/HomePhotos/closed.jpg";
+import { Accordion } from "react-bootstrap";
+import { Carousel } from "react-bootstrap";
+import { Card } from "react-bootstrap";
+import { useEffect, useState } from "react";
+import { useAuth } from "../Authentication/FirebaseAuth/FirebaseAuth";
+import "bootstrap/dist/css/bootstrap.min.css";
+import styles from "./MonitorPatients.module.css";
+import "./CommonPageStyling.css";
+import axios from "axios";
+import { ListGroup } from "react-bootstrap";
+
+function MonitorPatients(props) {
+
+    // //the following 5 const's are to test the ui without the backend:
```

Comment:
Do you need the commented text? They should not appear in the master branch

Reply:
You are right - I removed them!